src/hotspot/share/services/mallocTracker.hpp

```diff
@@ -191,10 +191,15 @@ class MallocMemorySnapshot : public ResourceObj {
     // copy is going on, because their size is adjusted using this
     // buffer in make_adjustment().
     ThreadCritical tc;
-    s->_all_mallocs = _all_mallocs;
-    for (int index = 0; index < mt_number_of_types; index ++) {
-      s->_malloc[index] = _malloc[index];
-    }
+    size_t total_mallocs;
```

Comment:
Can we rename from `total_mallocs` to `total_size`? When I looked at the code at first I thought we were counting the here the number of malloc operations.